sound/soc/sof/ipc4-topology.h

```diff
@@ -112,6 +114,29 @@ struct sof_ipc4_gtw_attributes {
 	uint32_t rsvd : 30;
 };
 
+/** struct sof_ipc4_alh_multi_gtw_cfg: ALH gateway cfg data
+ * @count: Number of single channels (valid items in mapping array)
+ * @alh_id: index of a single ALH channel aggregated
```

Comment:
it's very unclear if you are referring to ALH channels (DMA transfers that can handle multichannel data) or audio channels?

Reply:
> it's very unclear if you are referring to ALH channels (DMA transfers that can handle multichannel data) or audio channels?

This is actually alh_stream_id. sof_ipc4_alh_multi_gtw_cfg is used for sdw aggregation mode. I will update the comment,